docker/bin/run-prepare-release-assets

```diff
@@ -3,10 +3,9 @@
 set -o errexit
 set -o pipefail
 
-if [[ -z "${1}" ]]; then
-  echo "Please provide a target version, e.g. './docker.sh run-prepare-release-assets 1.2.3'"
-  exit 1
-fi
+SCRIPT_DIR="$(cd `dirname ${0}` && pwd)"
+
+source "${SCRIPT_DIR}/run-check-version"
```

Comment:
I think run-check-version will only check for component's package.json?

Reply:
Yes, when exec `./docker.sh run-prepare-release-components ${TARGET_VERSION}` it will compare the provided `TARGET_VERSION` with the version of `components`, which should be enough as the versions of:

- `components`
- `components-js/projects/components-wrapper`
- `components-js`
- `components-angular`
- `components-react`
- `components-vue`
etc.

are all in sync and will all be set to `TARGET_VERSION`.